crates/holochain_p2p/tests/tests/blocks.rs

```diff
@@ -325,6 +326,120 @@ async fn block_is_scoped_per_dna() {
         .unwrap());
 }
 
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
This test tests that outgoing get requests are dropped if I understand it correctly, i.e. that if Alice blocks Bob then a "get" from Alice to Bob won't succeed. This logic is not actually implemented yet in kitsune, it's currently pending in [this PR](https://github.com/holochain/kitsune2/pull/340). So it really only works because the insertion into the peer store is rejected. Which is still okay as a test but as written in the inline comment it's not really testing the blocks logic of outgoing messages. I'm not sure whether that's going to easily possible at all though so I'm probably fine with this test as it is.
However, shouldn't we also test the other way around, i.e. that a get request from Bob to Alice does not succeed if Bob had been blocked by Alice?

Reply:
> However, shouldn't we also test the other way around, i.e. that a get request from Bob to Alice does not succeed if Bob had been blocked by Alice?

Yes, that's a good one to add. That requires the next kitsune2 release, so I'd add that once that's done, alright?

Reply:
> Yes, that's a good one to add. That requires the next kitsune2 release

Does it? I didn't keep track of kitsune release, but the PR that contains the logic for blocking incoming messages has already been merged. And this test here is testing blocks of outgoing messages as I understand it.

Reply:
It's merged but not released yet.